modules/example/src/smithy4s/example/GetObjectOutput.scala

```diff
@@ -14,7 +14,7 @@ object GetObjectOutput extends ShapeTag.Companion[GetObjectOutput] {
   val hints: Hints = Hints.empty
 
   implicit val schema: Schema[GetObjectOutput] = struct(
-    ObjectSize.schema.required[GetObjectOutput]("size", _.size).addHints(smithy.api.Default(smithy4s.Document.fromDouble(0.0d)), smithy.api.Required(), smithy.api.HttpHeader("X-Size")),
+    ObjectSize.schema.required[GetObjectOutput]("size", _.size).addHints(smithy.api.Default(smithy4s.Document.fromDouble(0.0d)), smithy.api.HttpHeader("X-Size"), smithy.api.Required()),
```

Comment:
mmm, I wonder whether the CI is gonna scream about this : the ordering might be subject to the JVM you run. We'll see

Reply:
Failed, when running `sbt test` from nix shell, it produces old output. Reverting